HTTP2-Core.package/H2WebClient.class/instance/sendPriorKnowlege.contentBlock..st

```diff
@@ -2,7 +2,7 @@ sending
 sendPriorKnowlege: aRequest contentBlock: aContentBlock
```

Comment:
Die müssen wir unbedingt in sendPriorKnowledge (Betonung auf das "d") umbenennen :D